ui/css/itcss/components/network.scss

```diff
@@ -1,195 +0,0 @@
-@use "design-system";
-
-.network-component--disabled {
-  cursor: not-allowed;
-
-  .fa-caret-down {
-    opacity: 0;
-  }
-}
-
-.network-component.pointer {
-  border: 1px solid var(--color-border-muted);
-  border-radius: 82px;
-  padding: 6px 3px;
-  flex: 0 0 auto;
-  display: flex;
-
-  &.ethereum-network .menu-icon-circle div {
-    background-color: rgba(3, 135, 137, 0.7) !important;
-  }
-
-  &.goerli-test-network .menu-icon-circle div {
-    background-color: rgba(48, 153, 242, 0.7) !important;
-  }
-
-  &.sepolia-test-network .menu-icon-circle div {
-    background-color: rgba(207, 181, 240, 0.7) !important;
-  }
-
-  &.linea-test-network .menu-icon-circle div {
-    background-color: rgba(0, 0, 0, 0.7) !important;
-  }
-
-  &.localhost-network .menu-icon-circle div {
-    background-color: rgba(3, 135, 137, 0.7) !important;
-  }
-}
```

Comment:
Not used

![Screenshot 2025-02-07 at 11 42 59 AM](https://github.com/user-attachments/assets/1394c5d6-5147-4d94-8420-e5553cdfbb26)